dataflow/src/main/java/org/checkerframework/dataflow/cfg/AbstractCFGVisualizer.java

```diff
@@ -261,6 +261,33 @@ protected String loopOverBlockContents(
         }
     }
 
+    /**
+     * Format the given object into a String.
```

Comment:
How about `Format the given object as a String suitable for the output format, i.e. with format-specific characters escaped.`